oci/config/convert/runtime.go

```diff
@@ -123,31 +126,23 @@ func MutateRuntimeSpec(g rgen.Generator, rootfs string, image ispec.Image, manif
 
 	g.SetProcessUID(uint32(execUser.Uid))
 	g.SetProcessGID(uint32(execUser.Gid))
-	g.ClearProcessAdditionalGids()
```

Comment:
The thing is, the defaults given by the `runtime-tools/generate` is not defined anywhere. While I appreciate wanting to reduce the amount of code in our generation I don't really like relying on some random defaults that could change in the future (and might be a pain to detect).

But with this *especially* it's not a matter of the defaults from the generator, `Config.User` has semantic meaning with regards to supplementary groups. If a user specifies `100:100` this code won't touch the supplementary groups (which could be wrong and result in something unfortunate with privileged groups).